### PR TITLE
fix: revert import

### PR DESCRIPTION
**Pull request by jjenscodee**
## Description

(https://github.com/aica-technology/api/pull/60#discussion_r1377726768)

This PR solves the issue by updating the import. It also updates the version for api client.

## Review guidelines

Estimated Time of Review: 3 minutes

## Checklist before merging:

- [x] Confirm that the relevant changelog(s) are up-to-date in case of any user-facing changes

### CODE REVIEW

File: python/CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,12 @@
 
 Release Versions:
 
+- [1.0.1](#70)
```
Comment (domire8) [inline python/CHANGELOG.md:5]:
```suggestion
- [1.0.1](#101)
```

**Review by JasperTan97 — APPROVED** [review]
lgtm

File: python/CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,12 @@
...
 - [1.0.0](#100)
 
+## 1.0.1
+Version 1.0.1 fixes the import issue.
```
Comment (domire8) [inline python/CHANGELOG.md:9]:
```suggestion

Version 1.0.1 fixes a relative import issue.
```

**Review by domire8 — APPROVED** [review]
After that its okay for me, but @eeberhard knows more about relative imports in python

**Review by eeberhard — APPROVED** [review]
Since the python package is called `aica_api`, all modules need to be imported from the package namespace. The other change may have worked for local testing because the module was resolved on the python path, but that would not be the case after the package is installed to site packages.

**Pull request merged**